tools/cldr-code/src/main/java/org/unicode/cldr/util/Annotations.java

```diff
@@ -679,25 +758,48 @@ private static void writeList() {
         final UnicodeMap<Annotations> map = eng.getUnresolvedExplicitValues();
         Set<String> keys = new TreeSet<>(ChartAnnotations.RBC);
         map.keySet().addAllTo(keys);
-//        keys.add("👩🏻‍⚖");
+        //        keys.add("👩🏻‍⚖");
         for (String key : keys) {
-            System.out.println(Utility.hex(key, 4, "_").toLowerCase(Locale.ROOT)
-                + "\t" + key
-                + "\t" + map.get(key).getShortName()
-                + "\t" + Joiner.on(" | ").join(map.get(key).getKeywords()));
+            System.out.println(
+                Utility.hex(key, 4, "_").toLowerCase(Locale.ROOT) +
+                "\t" +
+                key +
+                "\t" +
+                map.get(key).getShortName() +
+                "\t" +
+                Joiner.on(" | ").join(map.get(key).getKeywords())
+            );
         }
         for (String s : Arrays.asList(
-            "💏", "👩‍❤️‍💋‍👩",
-            "💑", "👩‍❤️‍👩",
-            "👪", "👩‍👩‍👧",
-            "👦🏻", "👩🏿",
-            "👨‍⚖", "👨🏿‍⚖", "👩‍⚖", "👩🏼‍⚖",
-            "👮", "👮‍♂️", "👮🏼‍♂️", "👮‍♀️", "👮🏿‍♀️",
-            "🚴", "🚴🏿", "🚴‍♂️", "🚴🏿‍♂️", "🚴‍♀️", "🚴🏿‍♀️")) {
+            "💏",
```

Comment:
This one really breaks the intent, since it flattens out the logical structure.

Reply:
with this kind of formatter, that's bound to happen -- one approach would be to make the structure more explicit by defining separate arrays and then combining them, and that might be beneficial anyway in terms of separating the code from the data

if we really want a formatter to leave such code unchanged, it will have to be a very different kind of formatter, one that only enforces certain rules and otherwise doesn't change line-breaking or anything else. I don't know if such formatters exist...